analyze_headersMore_directives.gen.go

```diff
@@ -14,23 +15,23 @@
 
 package crossplane
 
-//nolint:gochecknoglobals
 var moduleHeadersMoreDirectives = map[string][]uint{
-	"more_clear_headers": {
-		ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
-	},
-	"more_clear_input_headers": {
-		ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
-	},
-	"more_set_headers": {
-		ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
-	},
-	"more_set_input_headers": {
-		ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
-	},
+    "more_clear_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
+    "more_clear_input_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
+    "more_set_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
+    "more_set_input_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
 }
 
+
 func HeadersMoreDirectivesMatchFn(directive string) ([]uint, bool) {
-	masks, matched := moduleHeadersMoreDirectives[directive]
-	return masks, matched
-}
+    m, ok := moduleHeadersMoreDirectives[directive]
+    return m, ok
+}
```

Comment:
Nit: Can the code generator output a newline at the end of the generated code? That way you don't have this annoying icon in the diff and anyone who tries to print the end of the file to their terminal will be a little happier.

Reply:
Applied: add a new line at the end of the generated code